### PR TITLE
updated categories to reflect quantity

### PR DESCRIPTION
**Pull request by sales-ls21**
## Description
Add description about what this pull request covers.
Per the referenced ticket, I updated the categories page to show the quantity of products in each category. 

## Number of Fixes
List of fixes included in this PR.
One, as per above. 

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#60 

## Expected Behavior
Describe the expected behavior.
The categories page should now reflect the number of items in each category.

## Steps to Test Solution
List steps to test your solution
`Git fetch`
`Git checkout`
(Optional, but recommended) -- delete database and migrations and rerun migrations
Navigate to `localhost://8000/categories` and make sure that categories now have a quantity displaying.

### CODE REVIEW

File: bangazon_website/bang_app/templates/categories.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	<div class="col-md-10 col-md-offset-2">
 		<h3><a href="{% url 'bang_app:product_list' id=category.id %}">{{ category.label }}</a></h3>
 		{% for p in category.products %}
+			<h5>Total Items in Category = ({{p.quantity}})</h5>
```
Comment (abbyfleming) [inline bangazon_website/bang_app/templates/categories.html]: Nice! Can this be refactored to just 
Category Name (Number)?
ie: School Supplies (3)

**Comment by sales-ls21** [comment]
It's harder to do given the way this is nested, but I can try it. I'm not
sure I can.

**Comment by abbyfleming** [comment]
Nice change. Right now it's displaying the quantity of the product. Can that be changed to the count of the items in the category. It might just be pulling in the count and displaying that.

**Comment by whitneycormack** [comment]
👍

**Comment by samphillips1879** [comment]
Looks good to go 👍

**Pull request merged**